### PR TITLE
WEBAPP-220: Use v3 of locations endpoint

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://integreat.atlassian.net/). 
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **WEBAPP**.

### CODE REVIEW

File: src/modules/endpoint/endpoints/cities.js [hunk 1]
```diff
@@ -17,15 +17,16 @@ const stripSlashes = (path: string): string => {
...
         name: _city.name,
         code: stripSlashes(_city.path),
         live: _city.live,
-        eventsEnabled: _city['ige-evts'] === '1',
-        extrasEnabled: true // todo
+        eventsEnabled: _city.events,
+        extrasEnabled: _city.extras
       }))
       .sort(_city => _city.name)
     return sortBy(cities, _city => _city.sortKey)
```
Comment (Schedulaar) [inline src/modules/endpoint/endpoints/cities.js]: Sorting should be different now - we get a new key `name_without_prefix` from the backend. That should be used for sorting now.

Comment (klinzo) [reply src/modules/endpoint/endpoints/cities.js]: yep, right.

File: src/modules/layout/containers/__tests__/__snapshots__/LocationLayout.spec.js.snap [hunk 2]
```diff
@@ -11,8 +11,8 @@ exports[`LocationLayout should show CategoriesToolbar if current route is catego
   }
   header={
     <Connect(Translate(LocationHeader))
-      isEventsEnabled={false}
-      isExtrasEnabled={false}
+      isEventsEnabled={undefined}
```
Comment (Schedulaar) [inline src/modules/layout/containers/__tests__/__snapshots__/LocationLayout.spec.js.snap]: I think these should not be undefined

Comment (klinzo) [reply src/modules/layout/containers/__tests__/__snapshots__/LocationLayout.spec.js.snap]: kind of, but we don't need them for LocationLayout... I find it way more clear to read the tests if we don't specify all atributes of models even if we don't need them. Although we have undefined in our snaps then

Comment (klinzo) [reply src/modules/layout/containers/__tests__/__snapshots__/LocationLayout.spec.js.snap]: kind of, but we don't need them for LocationLayout... I find it way more clear to read the tests if we don't specify all atributes of models even if we don't need them. Although we have undefined in our snaps then

**Pull request merged**